tox.ini

```diff
@@ -235,6 +235,7 @@ commands =
   python scripts/eachdist.py lint --check-only
 
 [testenv:docs]
+basepython: python3.9
```

Comment:
Can you add a TODO comment linking the sphinx issue?

Reply:
fixed